draft-ietf-rats-corim.md

```diff
@@ -1843,15 +1828,37 @@ The `cm` field is set based on the type of Conceptual Message inputted or to be
 
 All of the extracted and validated tags are loaded into an *appraisal context*.
 The Appraisal Context contains an internal representation of the inputted Conceptual Messages.
-The selected tags are mapped to the internal representation, making them suitable for appraisal processing.
+The selected tags are mapped to an internal representation, making them suitable for appraisal processing.
 
 [^issue] https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/96
 
-#### Reference and Endorsed Values Tranformation
+#### Reference Triples Tranformation {#sec-ref-trans}
+
+An available `rv` list entry {{sec-ir-ref-val}} is allocated.
+The Reference Values Triple {{sec-comid-triple-refval}} `environment-map` is copied to the `environment-map` for both the `rv` `condition` and `rv` `addition` ECTs.
+
+For each `measurement-map` entry in the measurements list, the i<sup>th</sup> `measurement-map` is copied to the i<sup>th</sup> `element-map` in the `elements` list of the `rv` `addition` ECT.
 
-The Reference Values ECT fields are populated as described in {{sec-phase1-trans}} and {{sec-ir-ref-val}}.
+The issuer of the Endorsement conceptual message is copied to the `ev` `addition` ECT authority field.
 
-The Endorsement Values ECT fields are populated as described in {{sec-phase1-trans}} and {{sec-ir-end-val}}.
+If the Endorsement conceptual message has a profile, the profile is copied to the `ev` `addition` ECT profile field.
+
+#### Endorsement Triples Tranformations {#sec-end-trans}
+
+Endorsement Triple Transformation :
+
+: An available `ev` entry {{sec-ir-end-val}} is allocated.
+The Endorsement Triple {{sec-comid-triple-endval}} `environment-map` is copied to the `environment-map` for both the `ev` `condition` and `ev` `addition` ECTs.
+
+: For each `measurement-map` entry in the measurements list, the i<sup>th</sup> `measurement-map` entry is copied to the i<sup>th</sup> entry in the `addition` ECT `elements` list.
+
+: The issuer of the Endorsement conceptual message is copied to the `ev` `addition` ECT authority field.
+
+: If the Endorsement conceptual message has a profile, the profile is copied to the `ev` `addition` ECT profile field.
+
+Conditional Endorsement Triple Transformation :
+
+Conditional Endorsement Series Triple Transformation :
```

Comment:
Are these TODOs (in which case, we should mark them explicitly and link to their associated issues)?  Or did you just forget to add the content?

Reply:
OK; stick a `[^issue] https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/XYZ` nearby then.